### PR TITLE
Theme Sidebar and Nav Items for Consistency with Central Dashboard UI

### PR DESCRIPTION
**Pull request by jenny-s51**
## Description

Themes PF sidebar and nav items to align with look and feel of Central Dashboard UI.

Before:
Screenshot 2024-11-19 at 17 01 05

After:
Screenshot 2024-11-26 at 1 06 30 PM

## How Has This Been Tested?

Visual changes only; run locally and verify that sidebar styling is consistent with Central Dashboard UI.

## Merge criteria:

- All the commits have been [_signed-off_](https://github.com/kubeflow/community/tree/master/dco-signoff-hook#signing-off-commits)  (To pass the `DCO` check)

- [x] The commits have meaningful messages; the author will squash them after approval or in case of manual merges will ask to merge with squash.
- [x] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [x] The developer has manually tested the changes and verified that the changes work.
- [x] Code changes follow the [kubeflow contribution guidelines](https://www.kubeflow.org/docs/about/contributing/).

If you have UI changes

- [x] The developer has added tests or explained why testing cannot be added.
- [x] Included any necessary screenshots or gifs if it was a UI change.
- [x] Verify that UI/UX changes conform the UX guidelines for Kubeflow.

### CODE REVIEW

**Comment by lucferbux** [comment]
Oh wow that's really nice, and I think it's more than enouth to give the kf look and feel.
Btw unrelated, but the main view might be a problem when rendered in kubeflow's central dashboard, we'll need to take a look. cc @Griffin-Sullivan @ederign once we have the central dashboard rendered we can ping jenny to see if there's any point of improvement in the MUI efforts. My guess is that if the ApplicationPage does not get the full height and has margins into the main view is gonna be a little bit out of place compared with other views, but we'll need to check it.

**Comment by lucferbux** [comment]
Either way this looks really great, thanks a lot @jenny-s51

File: clients/ui/frontend/src/shared/style/MUI-theme.scss [hunk 1]
```diff
@@ -55,6 +55,12 @@
   --mui-radio__input--Width: 10px;
   --mui-radio__input--Height: 10px;
 
+  // Sidebar from https://github.com/kubeflow/kubeflow/blob/4275d99754ac91f6cf5654b03824a73825e9fe55/components/centraldashboard/public/components/main-page.css#L7C1-L13C51 
+  --kf-central-primary-background-color: #0a3b71;
+  --kf-central-sidebar-default-color: #ffffff90;
```
Comment (jenny-s51) [inline clients/ui/frontend/src/shared/style/MUI-theme.scss:60]: Had to update this color from the default value used in Kubeflow to fix the a11y error - is https://github.com/kubeflow/kubeflow/issues a good place to log this in order to contribute it back? @lucferbux @Griffin-Sullivan ?

Comment (lucferbux) [reply clients/ui/frontend/src/shared/style/MUI-theme.scss]: Yes, that might be a good place to talk about that. Approving this though, we can continue the conversation there.

**Review by lucferbux — APPROVED** [review]
/lgtm

**Pull request merged**